src/geometry/manifold/vector_bundle/tangent.lean

```diff
@@ -141,12 +145,12 @@ does not pick wrong instances. In this section, we record the right instances fo
 them, noting in particular that the tangent bundle is a smooth manifold. -/
 
 section
-local attribute [reducible] tangent_space
 
 variables {M} (x : M)
 
-instance : module 𝕜 (tangent_space I x) := by apply_instance
 instance : inhabited (tangent_space I x) := ⟨0⟩
+instance : normed_add_comm_group (tangent_space I x) := by delta_instance tangent_space
+instance : normed_space 𝕜 (tangent_space I x) := by delta_instance tangent_space
```

Comment:
Did I already mention that I don't like these instances? :-)

Reply:
They should be gone once I merge master :-)